### PR TITLE
Todo List - Task sorting

### PR DESCRIPTION
**Pull request by tennjugu**
I have added a function that sorts tasks based on due dates and time (in events where the tasks have the same date).

### CODE REVIEW

File: src/form.js [hunk 1]
```diff
@@ -111,17 +118,19 @@ const newTodo = () =>{
...
-   const descriptionLabel = getFormDescription()
-   const descriptionTextarea = getDescriptionInput()
-   const dueDateLabel = getFormDueDate()
-   const dueDateInput = getDueDateInput()
-   const timeLabel = getFormTime()
-   const timeInput = getTimeInput()
-   const priorityLabel = getFormPriority()
-   const selectOption = getPriorityInput()
-   const formButton = getFormInputs()
+    const CloseBtn = getCancelBtn()
```
Comment (zizoh) [inline src/form.js]: Why is this named CloseBtn and not closeBtn or cancelBtn?

File: src/form.js [hunk 2]
```diff
@@ -134,26 +143,36 @@ const newTodo = () =>{
...
+        form.reset()
+    })
 }
 
 function addTask(){
     const mainSection = document.querySelector('.main')
     const tableLog = document.querySelector('#taskTable')
     const addBtn = document.querySelector('.addtask')
     addBtn.addEventListener('click',() => {
+        const form = document.getElementById('taskForm')
```
Comment (zizoh) [inline src/form.js]: A better name will be isForm or hasForm

File: src/form.js [hunk 3]
```diff
@@ -134,26 +143,36 @@ const newTodo = () =>{
...
+        if(form){
+            mainSection.removeChild(form)
+            console.log('hi')
+        }
         if(mainSection.contains(tableLog)){
             tableLog.style.display = 'none'
             newTodo()
+            console.log('hello')
```
Comment (zizoh) [inline src/form.js]: Please remove this log

File: src/form.js (hunk 1 above)
Comment (tennjugu) [reply src/form.js]: > Why is this named CloseBtn and not closeBtn or cancelBtn?
I've rectified it.

**Review by zizoh — APPROVED** [review]

**Pull request merged**